### ISSUE TITLE
Gutenex.set_page/2 doesn't allow you to generate data on new page

### ISSUE THREAD

**Issue by kennellroxco** [SenecaSystems/gutenex#10]
I can't seem to get the set_page/2 function to work properly. I'm not sure whether I am using it incorrectly or if it isn't working properly, but I'm basically trying to iterate through various data sets and create an additional page for each data set within the same PDF file. Any guidance you could provide on how to create a multi-page PDF would be greatly appreciated!

**Comment by tyre** [comment]
Oh interesting. What happens when you do this? I may need to add a function for actually creating a new page

**Comment by tyre** [comment]
Yeah there is a function to add a new page (https://github.com/SenecaSystems/gutenex/blob/master/lib/gutenex/pdf.ex#L11) but I don't think it is exposed anywhere. Certainly needed!
